repositories/MerchStoreRepository.ts

```diff
@@ -41,6 +35,12 @@ export class MerchCollectionRepository extends BaseRepository<MerchandiseCollect
   public async deleteMerchCollection(collection: MerchandiseCollectionModel): Promise<void> {
     await this.repository.remove(collection);
   }
+
+  public getBaseFindManyQuery(): SelectQueryBuilder<MerchandiseCollectionModel> {
```

Comment:
nit: since this code refactoring wasn't part of the initial issue, imo it'd be good to call it out in the PR (just so I'm not confused as to why this file was changed)

Reply:
Good call, yeah this change was a quick one for refactoring common logic. Will call those out next time